### PR TITLE
update sdk and ibc-go

### PR DESCRIPTION
**Pull request by jhernandezb**

### CODE REVIEW

**Review by yubrew — COMMENTED** [review]
why do some of these go.mod version changes do not change hash? ex: 
```
github.com/cpuguy83/go-md2man/v2 v2.0.1/go.mod h1:tgQtvFlXSQOSOSIRvRPT7W67SCa46tRHOmNcaadrF8o=
github.com/cpuguy83/go-md2man/v2 v2.0.2/go.mod h1:tgQtvFlXSQOSOSIRvRPT7W67SCa46tRHOmNcaadrF8o=
```

**Comment by jhernandezb** [comment]
> 

I don't think that dependency was updated 
Screen Shot 2022-06-22 at 10 42 25 AM

File: x/alloc/types/events.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package types
 
 const (
-	EventTypeClaim         = "fund_fairburn_pool"
-	AttributeValueCategory = ModuleName
+	EventTypeFundFairburnPool = "fund_fairburn_pool"
```
Comment (yubrew) [inline x/alloc/types/events.go:4]: 50% fairburn goes to this sdk pool instead of community?

**Review by yubrew — APPROVED** [review]
👍

**Pull request merged**